tests/e2e/tutorials/tutorialBase.js

```diff
@@ -225,6 +225,18 @@ class TutorialBase {
     return;
   }
 
+  async waitForStudyUnlocked(studyId, timeout = 10000) {
+    const start = new Date().getTime();
+    while ((new Date().getTime())-start < timeout) {
+      await utils.sleep(1000);
```

Comment:
the ``sleep`` here will produce a timeout if the value of ``timeout`` is smaller that ``1000``
do something like `` timeout/100``